### DISCUSSION TITLE
Im Medienpool werden keine Vorschaubilder mehr angezeigt

### DISCUSSION THREAD

**Discussion by tyrant88** [FriendsOfREDAXO/.github#24] [Q&A] [answered]
Nach einem Update werden im Medienpool keine Vorschaubilder angezeigt. 
Was kann man tun? Wie bekommt man die wieder angezeigt?

**Comment by danspringer** [comment, marked as answer]
Wenn YRewrite installiert ist, dieses AddOn reinstallieren, bzw, unter "Setup" in YRewrite die `.htaccess-Datei` neu setzen lassen.

**Comment by tyrant88** [comment]
Eventuell wurde auch ein Wechsel vom Apache auf nginx vorgenommen. Da greift dann die .htaccess gar nicht.
Stattdessen sollte ein Konfiguration gemaß: https://github.com/yakamara/yrewrite#nginx-konfiguration-f%C3%BCr-yrewrite vorgenommen werden.

**Reply by gharlan** [reply]
Eher gemäß https://github.com/yakamara/yrewrite#nginx-konfiguration-für-yrewrite (die Syntax in nginx ist anders).

**Reply by gharlan** [reply]
Ha, da hat jemand seinen Beitrag editiert ;)
